tests/transformers/test_transformers_model_export.py

```diff
@@ -3924,3 +3632,65 @@ def test_text_generation_task_chat_serve(text_generation_pipeline):
         and output_dict["usage"]["completion_tokens"] < 10
     )
     assert output_dict["usage"]["prompt_tokens"] < 20
+
+
+HF_COMMIT_HASH_PATTERN = re.compile(r"[a-z0-9]{40}")
+
+
+@pytest.mark.parametrize(
+    ("model_fixture", "input_example", "components"),
+    [
+        ("text2text_generation_pipeline", "What is MLflow?", {"tokenizer"}),
+        ("text_generation_pipeline", "What is MLflow?", {"tokenizer"}),
+        (
+            "small_vision_model",
+            image_url,
+            {"feature_extractor", "image_processor"}
+            if IS_NEW_FEATURE_EXTRACTION_API
+            else {"feature_extractor"},
+        ),
+        (
+            "component_multi_modal",
+            {"text": "What is MLflow?", "image": image_url},
+            {"image_processor", "tokenizer"}
+            if IS_NEW_FEATURE_EXTRACTION_API
+            else {"feature_extractor", "tokenizer"},
+        ),
+        ("fill_mask_pipeline", "The quick brown <mask> jumps over the lazy dog.", {"tokenizer"}),
+        ("whisper_pipeline", read_raw_audio_file, {"feature_extractor", "tokenizer"}),
```

Comment:
Note: not `read_raw_audio_file()` because pytest uses the serialized parameter value for naming tests and giving the raw bytes makes it so messy.